packages/boxel-ui/addon/src/components/resizable-panel-group/index.gts

```diff
@@ -155,11 +176,29 @@ export default class ResizablePanelGroup extends Component<Signature> {
     this.listPanelContext.set(id, {
       defaultLengthFraction: context.defaultLengthFraction,
       lengthPx: context.lengthPx,
+      minLengthPx: context.minLengthPx,
     });
 
+    this.calculatePanelRatio();
+
     return id;
   }
 
+  @action
```

Comment:
I think this doesn’t need to be `@action` since it looks like it’s only called from internal functions vs the template